lib/benchee/formatters/csv.ex

```diff
@@ -27,10 +29,10 @@ defmodule Benchee.Formatters.CSV do
   a CSV, but also already writes it to a file defined in the initial
   configuration under `[formatter_options: [csv: [file: \"my.csv\"]]`
   """
-  @spec output(Benchee.Suite.t) :: Benchee.Suite.t
+  @spec output(Suite.t) :: Suite.t
   def output(suite)
-  def output(suite = %{configuration:
-              %{formatter_options: %{csv: %{file: filename}}}}) do
+  def output(suite = %Suite{configuration:
+              %Configuration{formatter_options: %{csv: %{file: filename}}}}) do
     suite
     |> format
     |> write_csv_to_file(filename)
```

Comment:
maybe a different thing, but I actually thought about changing it so that it all ends up in one file again... people should be able to get what they want with their excel or whatever powers. And it's really not that much data, that we need to discern it. Topic for a different PR though.